composer.json

```diff
@@ -29,37 +29,36 @@
     }
   ],
   "require": {
-    "php": "^7.1.3",
+    "php": "^7.4.3",
     "ext-json": "*",
-    "beberlei/assert": "^3.3.0",
-    "ergebnis/json-normalizer": "^0.13.0 || ^1.0.2",
-    "mimmi20/json-class": "~2.0.16",
-    "mimmi20/ua-browser-type": "^7.0.15",
-    "mimmi20/ua-device-type": "^7.1.14",
-    "monolog/monolog": "^1.26.0 || ^2.2.0",
+    "beberlei/assert": "^3.3.1",
+    "mimmi20/json-class": "^4.0.0",
+    "mimmi20/ua-browser-type": "^8.0.5",
+    "mimmi20/ua-device-type": "^8.0.5",
+    "monolog/monolog": "^2.3.5",
     "seld/jsonlint": "^1.8.3",
-    "symfony/console": "^4.4.18 || ^5.2.1",
-    "symfony/finder": "^4.4.18 || ^5.2.1",
-    "twig/twig": "^2.13.1 || ^3.2.1"
+    "symfony/console": "^v5.4.0",
+    "symfony/finder": "^v5.4.0",
+    "twig/twig": "^v3.3.4"
```

Comment:
Just noticed this - why do we need twig? :thinking:

Reply:
We have a console command to rewrite the files in the resources directory. Twig is used to render the files to keep the structure we have.